DigitalOffice/Tests/CompanyServiceTests/Repositories/CompanyRepositoryTests.cs

```diff
@@ -0,0 +1,57 @@
+﻿using LT.DigitalOffice.CompanyService.Database;
+using LT.DigitalOffice.CompanyService.Repositories;
+using LT.DigitalOffice.CompanyService.Database.Entities;
+using Microsoft.EntityFrameworkCore;
+using NUnit.Framework;
+using System;
+using LT.DigitalOffice.CompanyService.Repositories.Interfaces;
+using LT.DigitalOffice.CompanyServiceUnitTests.UnitTestLibrary;
+
+namespace LT.DigitalOffice.CompanyServiceUnitTests.Repositories
+{
+    public class CompanyRepositoryTests
+    {
+        private CompanyServiceDbContext dbContext;
+        private ICompanyRepository repository;
+
+        private DbCompany dbCompanyInDB;
+
+        [OneTimeSetUp]
+        public void OneTimeSetUp()
+        {
+            var dbOptions = new DbContextOptionsBuilder<CompanyServiceDbContext>()
+                                    .UseInMemoryDatabase(databaseName: "InMemoryDatabase")
+                                    .Options;
+            dbContext = new CompanyServiceDbContext(dbOptions);
+            repository = new CompanyRepository(dbContext);
+
+            dbCompanyInDB = new DbCompany
+            {
+                Id = Guid.NewGuid(),
+                Name = "Lanit-Tercom",
+                IsActive = true
+            };
+
+            dbContext.Companies.Add(dbCompanyInDB);
+            dbContext.SaveChanges();
+        }
+
+        [TearDown]
+        public void Clear()
+        {
+            if (dbContext.Database.IsInMemory())
+            {
+                dbContext.Database.EnsureDeleted();
+            }
+        }
+
+        [Test]
+        public void SuccessfulGetCompanyById()
+        {
+            var result = repository.GetCompanyById(dbCompanyInDB.Id);
+
+            var resultCompany = dbContext.Companies.FirstAsync().Result;
+            SerializerAssert.AreEqual(result, resultCompany);
```

Comment:
Two results? Rename variables and swap arguments

Reply:
Fixed.